python/ray/autoscaler/commands.py

```diff
@@ -386,7 +384,7 @@ def exec_cluster(config_file,
         port_forward (int or list[int]): port(s) to forward
     """
     assert not (screen and tmux), "Can specify only one of `screen` or `tmux`."
-
+    assert run_env in RUN_ENV_TYPES
```

Comment:
can you please print an actionable error message?

Reply:
Will do!

Reply:
Actually, I'm removing this check in favor of using `click.Choice` which is how to specify a select number of args.